src/get-own-property-symbols.js

```diff
@@ -258,7 +258,7 @@
           descriptor.value = function () {
             var
               str = toString.call(this),
-              tst = this[Symbol.toStringTag]
+              tst = typeof this !== "undefined" ? this[Symbol.toStringTag] : this
```

Comment:
`this` could also be `null`, so you may want to do:
```suggestion
              tst = this == null ? this : this[Symbol.toStringTag];
```